src/core/ext/filters/client_channel/lb_policy/rls/rls.cc

```diff
@@ -1361,11 +1368,10 @@ RlsLb::Cache::Entry::OnRlsResponseLocked(
 //
 
 RlsLb::Cache::Cache(RlsLb* lb_policy) : lb_policy_(lb_policy) {
-  Timestamp now = Timestamp::Now();
   lb_policy_->Ref(DEBUG_LOCATION, "CacheCleanupTimer").release();
-  GRPC_CLOSURE_INIT(&timer_callback_, OnCleanupTimer, this, nullptr);
-  grpc_timer_init(&cleanup_timer_, now + kCacheCleanupTimerInterval,
-                  &timer_callback_);
+  cleanup_timer_handle_ =
```

Comment:
Suggest moving this code into a separate private method called `StartCleanupTimer()`, so that we can just call it from both here and from the end of `OnCleanupTimer()` (line 1460).

Reply:
Done